opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -981,7 +982,7 @@ export const userIdDeleteRelation = async (context, user, userId, toId, relation
 };
 
 export const userDeleteOrganizationRelation = async (context, user, userId, toId) => {
-  if (!isUserHasCapability(user, SETTINGS_SET_ACCESSES) && isUserHasCapability(user, VIRTUAL_ORGANIZATION_ADMIN)) {
+  if (!isAdminWithoutSettingsSetAccessCapability(user)) {
```

Comment:
should be `if (isAdminWithoutSettingsSetAccessCapability(user))`, without the `!`